pom.xml

```diff
@@ -51,6 +51,7 @@
         <junit.testkit.version>1.7.1</junit.testkit.version>
         <slf4j.version>1.7.30</slf4j.version>
         <hamcrest.version>2.2</hamcrest.version>
+        <spring.version>2.5.3</spring.version>
```

Comment:
I think I would opt for labeling this as the spring-boot version over the Spring version